### DISCUSSION TITLE
Generating a texi from README.org without a export_file_name

### DISCUSSION THREAD

**Discussion by DamienCassou** [emacscollective/borg#123] [Ideas]
The function `borg-maketexi` requires the presence of `export_file_name` in the README.org file. Unfortunately, this property might be used by the author of the README for another export (e.g., the html export). Moreover, org-mode says that this property is optional:

    ‘EXPORT_FILE_NAME’
       The name of the output file to be generated.  Otherwise, Org
       generates the file name based on the buffer name and the extension
       based on the back-end format.

Would you accept a patch making this property optional?

**Comment by tarsius** [comment]
> The function `borg-maketexi` requires the presence of `export_file_name` in the README.org file.

I don't think that is true.

**Reply by DamienCassou** [reply]
You are right. My problem is different. The README.org has "package.html" as value for `export_file_name` and `package.html` is an existing file and is tracked by the git repository. This prevents `borg-maketexi` from generating the unrelated `package.texi`.

**Reply by tarsius** [reply]
Tadaaa!
Fixed :grinning:

**Reply by DamienCassou** [reply]
Excellent, thank you very much!

**Reply by tarsius** [reply]
You are welcome!
